### PR TITLE
DT-6033 Raasepori configuration

### PR DESCRIPTION
**Pull request by sharhio**
## Proposed Changes

  - Map zoom level configurable 
  - AppBarLink language configurable
  - Raasepori instance configurations

## Pull Request Check List

  - A reasonable set of unit tests is included
  - Console does not show new warnings/errors
  - Changes are documented or they are self explanatory
  - This pull request does not have any merge conflicts
  - All existing tests pass in CI build
  - Code coverage does not decrease (unless measured incorrectly)

## Review

  - Read and verify the code changes
  - Test the functionality by running the UI locally with all popular browsers available in your platform
  - Check that the implementation matches the design, when such one is defined in a Jira issue
  - Merge the pull request

### CODE REVIEW

File: app/component/MainMenu.js [hunk 1]
```diff
@@ -16,6 +16,11 @@ import intializeSearchContext from '../util/DTSearchContextInitializer';
 
 function MainMenu(props, { config, intl }) {
   const [countries, setCountries] = useState(props.countries);
+  const appBarLinkHref =
+    config.appBarLink.alternativeHref &&
```
Comment (vesameskanen) [inline app/component/MainMenu.js]: Tämä ehkä menee siistimmin näin:

Href  = config.appBarLink.alternativeHref?.[props.currentLang] || config.appBarLink.href;

File: app/component/MainMenu.js [hunk 2]
```diff
@@ -117,12 +122,12 @@ function MainMenu(props, { config, intl }) {
           ))}
         {config.appBarLink &&
           config.appBarLink.name &&
```
Comment (vesameskanen) [inline app/component/MainMenu.js]: tässäkin vois refaktoroida vähän ja käyttää js optional chaining

File: app/configurations/config.raasepori.js [hunk 3]
```diff
@@ -16,6 +16,10 @@ export default configMerger(walttiConfig, {
   appBarLink: {
     name: 'Bosse',
     href: 'https://raasepori.fi/bosse',
+    alternativeHref: {
+      sv: 'https://raseborg.fi/bosse',
+      en: 'https://raasepori.fi/bosse',
```
Comment (vesameskanen) [inline app/configurations/config.raasepori.js]: en versio on turha, fallback fi versioon tekee saman

Comment (sharhio) [reply app/configurations/config.raasepori.js]: Jees, hyvä pointti, tässä oli alkuun hiukan arvontaa mikä linkki tulee en valinnalla.

**Review by vesameskanen — APPROVED** [review]

**Pull request merged**